pr_demo.py

```diff
@@ -1,3 +1,5 @@
 """
 Stock file to be used in PR demo.
-"""
\ No newline at end of file
+"""
+def new_method():
```

Comment:
why did you choose this name? we might want to think about a more descriptive name.